src/Bicep.Decompiler/BicepDecompiler.cs

```diff
@@ -101,6 +103,24 @@ public async Task<DecompileResult> Decompile(Uri entryJsonUri, Uri entryBicepUri
             PrintFiles(workspace));
     }
 
+    public string? DecompileJsonValue(string jsonInput, DecompileOptions? options = null)
+    {
+        var workspace = new Workspace();
+        var decompileQueue = new Queue<(Uri, Uri)>();
```

Comment:
nit: Remove?

Reply:
It's used here:

        private ResultAndTelemetry? TryConvertFromJsonValue(StringBuilder output, string json, string decompileId, bool queryCanPaste)
        {
            var singleFileResolver = new SingleFileResolver(JsonDummyUri, json);

            var decompiler = new BicepDecompiler(this.bicepCompiler, singleFileResolver);
            var pasteType = PasteType_JsonValue;
            var options = GetDecompileOptions(pasteType);
            var bicep = decompiler.DecompileJsonValue(json, options);